concepticondata/conceptlists/BeijingDaxue-1964-905.tsv

```diff
@@ -611,7 +611,7 @@ BeijingDaxue-1964-905-609	609	藏(收存)	305	verb	daily actions	四、日常操
 BeijingDaxue-1964-905-610	610	說	305	verb	communication	五、交際、事務、人事	動詞	speak	shuō	1623	SPEAK	说
 BeijingDaxue-1964-905-611	611	閑淡	306	verb	communication	五、交際、事務、人事	動詞	talk, make smalltalk	xián dàn	2708	TALK	闲澹
 BeijingDaxue-1964-905-612	612	問	306	verb	communication	五、交際、事務、人事	動詞	ask	wèn	1309	ASK (INQUIRE)	问
-BeijingDaxue-1964-905-613	613	理	307	verb	communication	五、交際、事務、人事	動詞	manage	lǐ			理
+BeijingDaxue-1964-905-613	613	理	307	verb	communication	五、交際、事務、人事	動詞	manage	lǐ	3073	MANAGE	理
```

Comment:
Not sure whether the intended meaning her fits ID 3073.

Reply:
Difficult to say, but we can leave this for the time being.